config/initializers/assets.rb

```diff
@@ -9,4 +9,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
-Rails.application.config.assets.precompile += ["authentication.css"]
+Rails.application.config.assets.precompile += ["authentication.css", "test.js", "test.css"]
```

Comment:
should we not make this dependent on `Rails.env.test?`

Reply:
Good catch, I think we can even remove this (during development and test there's no need to precompile anything).

Reply:
I think we could add this to `environments/test.rb`.

Reply:
What should I do with this?

Reply:
I think we can remove both (`test.js` and `test.css`) and everything should work, can you double check that?

Reply:
We can't remove.

```
Asset was not declared to be precompiled in production.
Add `Rails.application.config.assets.precompile += %w( test.js )` to `config/initializers/assets.rb` and restart your server
```

Reply:
Ouch, yes, we can move it to `environments/test.rb` then?